components/README.md

```diff
@@ -11,3 +11,13 @@ yarn add @brightlayer-ui/angular-components
 ```
 
 > **NOTE**: This install command will not work until the component libraries are published to NPM. Until then you will need to follow the manual linking instructions.
+
+## Peer Dependencies
```

Comment:
I would remove this table from the readme since it's duplicating info found in the package.json & we already have a compatibility table in this file.